portal/app/core/views/base.py

```diff
@@ -35,34 +35,34 @@ def index(request):
         template = 'partials/main_lab.html'
     elif request.user.powernetuser.type == PowernetUserType.FARM:
 
-        # querying power consumption data
-        # pen_power also used for bottom row control fan power
-        last_object = FarmData.objects.filter(farm_device_id = 17).latest('timestamp')
-        serialized_farm_data = FarmDataSerializer(last_object).data
-        pen_power = json.dumps(main_farm_parser(serialized_farm_data))
+        try:
+            # querying power consumption data
+            # pen_power also used for bottom row control fan power
+            last_object = FarmData.objects.filter(farm_device_id = 17).latest('timestamp')
+            serialized_farm_data = FarmDataSerializer(last_object).data
+            pen_power = json.dumps(main_farm_parser(serialized_farm_data))
```

Comment:
this seems real weird to me. It seems like you are taking the model object, serializing it via `FarmDataSerializer`, then parsing it, then serializing it again?
Seems like should be able to skip a step here right?

Reply:
you're right, so remove the json.dumps?

Reply:
I'll let you play around with it and figure it out. Should be fairly simple, but let me know if you need some pointers.

Reply:
Done